src/components/ContentHighlights/ZeroStateHighlights.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {
+  Card, Button, useMediaQuery, breakpoints,
+} from '@edx/paragon';
+import PropTypes from 'prop-types';
+import { Add } from '@edx/paragon/icons';
+import cardImage from './data/images/ContentHighlightImage.svg';
+import ZeroStateCardImage from './ZeroStateCardImage';
+import ZeroStateCardText from './ZeroStateCardText';
+import ZeroStateCardFooter from './ZeroStateCardFooter';
+
+const ZeroStateHighlights = ({ cardClassName }) => {
+  const isExtraSmall = useMediaQuery({ maxWidth: breakpoints.large.minWidth });
+
+  return (
+    <Card className={cardClassName} style={{ width: isExtraSmall ? '100%' : '949px' }}>
```

Comment:
We could leave the `Card` as taking up 100% width of its container, but use `Col` to control its width as various breakpoints.

```jsx
<Col xs={12} sm={8} lg={5}>
  <Card>
    {/* ... */}
  </Card>
</Col>
```